opa/security/opa_security.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="module_opa_category" model="ir.module.category">
+        <field name="name">Open Academy</field>
+        <field name="description">Helps you manage all your courses</field>
+    </record>
+
+    <record id="group_opa_manager" model="res.groups">
```

Comment:
Move all the records related to `res.groups` to `security/res_groups.xml`

Reply:
Done